woocommerce/woocommerce-android
# [UOE] Handle multiple shipping lines edit

Closes: #6653


Twin iOS PR: https://github.com/woocommerce/woocommerce-ios/pull/7197

### Description

This PR adds support for removing only the first shipping line (previously removing one line removed all of them) and showing the total of shipping lines in the payment section.

### Testing instructions

Taken from iOS PR:
> - Create an order with multiple shipping lines on Core
> - Go to the order in the app and edit the shipping lines
> - See that only the first shipping line is presented and can be edited.
> - Remove that shipping line
> - See that the others shipping lines remain unaltered


### Images/gif


https://user-images.githubusercontent.com/5845095/177825609-a141d020-484a-4051-868d-876059486fac.mov
